general: added the --exclude flag

jperezdealgaba opened the pull request:
added capabilities of excluding results from the ZAP results separated by a blank space.

Yadnyawalkya commented on README.md:
```diff
@@ -36,6 +41,15 @@ or
 
+## Exclusions
+
+In order to exclude findings from the results, the user can remvoe them by using the `--exclude` flag:
```
nitpick, typo: replace `remvoe` by `remove`

jperezdealgaba replied:
Fixed!

jeremychoi commented on rapidast_parser.py:
```diff
@@ -59,11 +80,30 @@ def check_names(path):
+                if k in zap_alert_properties:
+                    if k in alert:
+                        if v == alert[k]:
+                            in_exception = True
```
Break should be needed after this. If there are multiple exclusions, in_exception will be changed back to False by another exclusion.

jperezdealgaba replied:
Good catch. Added!

jeremychoi commented on README.md:
````diff
@@ -36,6 +41,15 @@ or
+
+In order to exclude findings from the results, the user can remove them by using the `--exclude` flag:
+```
+python3 rapidast_parser.py --tool garak --file report-hitlog.jsonl --exclude pluginid:40018,response-header:"HTTP/1.1 401 Unauthorized" pluginid:40018,response-header:"HTTP/1.1 41231 Unauthorized"
````
Just a few comments: 
1. It should be '--tool zap'?
2. how about another a bit different example from the first one, instead of "pluginid:40018,response-header:"HTTP/1.1 41231 Unauthorized"? e.g. "pluginid:10000,method:POST"

jperezdealgaba replied:
Fixed!

jperezdealgaba committed:
general: added the --exclude flag
added capabilities of excluding results from the ZAP results separated by a blank space

jeremychoi commented:
LGTM now
